collab-folder/src/core/relation.rs

```diff
@@ -28,6 +28,60 @@ impl ViewRelations {
     })
   }
 
+  /// Disconnect relation between parent_id and view_id
+  pub fn disconnect_child_with_txn(
```

Comment:
Why separating the `move` function into `disconnect` and `connect`?

Is it used for deleting?